cli/cmd/get_event.go

```diff
@@ -143,11 +137,8 @@ func init() {
 
 	getEventParams.NumOfPages = getEventCmd.Flags().IntP("num-of-pages", "", 1,
 		"Number of pages that should be returned (Default 1).")
-}
 
-func setParameterValue(value string, defaultValue string) string {
-	if len(value) == 0 {
-		return defaultValue
-	}
-	return value
+	getEventParams.Watch = getEventCmd.Flags().BoolP("watch", "w", false, "Print event stream")
+
+	getEventParams.WatchTime = getEventCmd.Flags().Int("watch-time", math.MaxInt32, "Timeout (in seconds) used for the --watch flag")
```

Comment:
is `MaxInt32` best practice for such timeouts? I think something in the order of 2-5 minutes makes more sense.

Reply:
If you do not specify a timeout I would expect such a flag to watch forever... But that's just my expectation. I'm also happy to change it to default to a couple of minutes.
Maybe we collect a third opinion? ( @bacherfl ?)

Reply:
That's a totally valid point :+1:

Reply:
I would also expect the command to wait forever if I do not specify a timeout

Reply:
Then let's do it like that ;) Thanks Florian and Bernd!